dockerfile_functions.sh

```diff
@@ -770,9 +770,7 @@ EOI
 }
 
 # Turn on JVM specific optimization flags.
-# Hotspot container support = https://bugs.openjdk.java.net/browse/JDK-8189497
-# OpenJ9 container support = https://www.eclipse.org/openj9/docs/xxusecontainersupport/
-# OpenJ9 Idle tuning = https://www.eclipse.org/openj9/docs/xxidletuninggconidle/
+# Hotspot@JDK9: https://bugs.openjdk.java.net/browse/JDK-8189497
 print_java_options() {
```

Comment:
this method is not doing anything, as jdk9 is not supported. But probably useful to just keep